superset/viz.py

```diff
@@ -68,19 +68,22 @@ def __init__(self, datasource, form_data):
         self.status = None
         self.error_message = None
 
-    def get_fillna_for_type(self, col_type):
+    def get_fillna_for_col(self, col):
         """Returns the value for use as filler for a specific Column.type"""
-        if col_type:
-            if col_type == 'TEXT' or col_type.startswith('VARCHAR'):
-                return ' NULL'
+        if col:
+            if col.is_string:
+                return 'NULL'
```

Comment:
The space was done on purpose for having NULL entries before other values after sorting

Reply:
gotcha added it back